src/world/Actor/BNpc.cpp

```diff
@@ -306,6 +311,10 @@ void Sapphire::Entity::BNpc::update( int64_t currTime )
 
     case BNpcState::Idle:
     {
+      // passive mobs should ignore players unless aggro'd
+      if( m_aggressionMode == 1 )
```

Comment:
we should enum these at some point

Reply:
in fact, i think there's already enums for these in exds

Reply:
I can't find it in the exds, I know it was there at one point but looked before making the PR and couldn't find it

maybe i'm blind?